### DISCUSSION TITLE
Hand Model in Unity with Reverb G2

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/MixedRealityToolkit-Unity#9127] [Q&A] [answered]
I'm using the Reverb G2 and was wondering how do I get a hand model in Unity with MRTK. I've already racked my brain about a model of the controller, without success and now I'm looking for a way to replace the default cursor with hand models.
There are some tutorials and possibilities with the MRTK for the Oculus quest. But how does that work with the reverb?

**Comment by david-c-kline** [comment]
@SaschaFFM,

The Reverb G2 does not support hand tracking. You may be able to have success attaching an UltraLeap Leap Motion controller to the front of the headset and use the MRTK's Leap Motion data provider.

Please note that this is not something that I have personally attempted and it may not work as desired.

**Reply by ghost** [reply]
Hello, no, I didn't mean hand tracking, I meant a model from a hand. I probably wrote about the quest, therefore, the misunderstanding. I just meant that you can find good tutorials for the quest how to integrate a hand model in unity. And I´m still searching  for an easy way to get a model from the controller in unity. With steamvr i have a perfect model in unity and in the demo scene in the openxr runtime ther is also a controller model. So there must be a controller model on my pc but how can i get this in MRTK in Unity?...

**Reply by david-c-kline** [reply]
Ahh i see. Thanks for the update.

**Comment by keveleigh** [comment, marked as answer]
I recently merged https://github.com/microsoft/MixedRealityToolkit-Unity/pull/9550 which adds runtime controller models with XR SDK. Currently, controller models while running in the editor is still blocked by https://github.com/microsoft/MixedRealityToolkit-Unity/issues/3589
